hugo/layouts/shortcodes/youtube.html

```diff
@@ -0,0 +1,34 @@
+{{- $id := .Get "id" | default (.Get 0) -}}
+{{- $title := .Get "title" | default "" }}
+{{- $image := .Get "image" | default "" }}
+{{- $url := "https://www.youtube.com/watch?v={{ $id }}" }}
```

Comment:
This is not working: you now go to the url /$id  . Use printf to use variables inside variables